.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
```

Comment:
Since you are using JavaScript for this milestone, you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository.

Kindly update your `linters.yml` file and replace it with the one from the [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) Linters Config repository.

![image](https://user-images.githubusercontent.com/38088818/167582683-57ee2c36-0b50-4a9c-9042-a74e97e9e88b.png)

- Please make sure your ESlint linter checks are passing after updating the config files.